### DISCUSSION TITLE
Very slow

### DISCUSSION THREAD

**Discussion by craigbilner** [dagger/dagger#5288] [Help]
When running the basic setup from the examples on our repo it takes several minutes to "transfer" (20s even when cached) and several minutes to copy (regardless of any file changes?), is there a nice way to debug/understand where bottlenecks are with the terminal UI?

**Comment by craigbilner** [comment]
EDIT: I was using the UI terminal with `--debug`, if I just run `node foo.mjs` I can see the size of things being uploaded/transferred

**Comment by marcosnils** [comment]
:wave: can you please provide more information about your environment? 
what system are you running Dagger on? 
how big is your repo?
what SDK are you using?

Can you enable log outputs in your dagger run and paste them here? :pray:
